src/DIRAC/Core/Utilities/Os.py

```diff
@@ -147,5 +147,4 @@ def sourceEnv(timeout, cmdTuple, inputEnv=None):
 
 
 def which(executable):
```

Comment:
This looks to be used only by VOMS.py. Maybe modify VOMS module and add the `deprecated` decorator to this function.

Reply:
Done